cypress/support/customCommands/basicOperations/index.js

```diff
@@ -50,6 +50,15 @@ Cypress.Commands.add("checkIfDataAreNotNull", data => {
 Cypress.Commands.add("checkIfElementIsVisible", element => {
   cy.get(element).should("be.visible");
 });
+Cypress.Commands.add("checkIfElementIsNotVisible", element => {
+  cy.get(element).should("not.be.visible");
+});
+Cypress.Commands.add("checkIfElementExist", element => {
+  cy.get(element).should("exist");
+});
+Cypress.Commands.add("checkIfElementNotExist", element => {
+  cy.get(element).should("not.exist");
+});
```

Comment:
If you check cypress docs, you will find "There's no reason to add this level of complexity when you're only wrapping a couple commands.

Don't do things like:

cy.clickButton(selector)
 .shouldBeVisible()
This first custom command is wrapping cy.get(selector).click(). Going down this route would lead to creating dozens or even hundreds of custom commands to cover every possible combination of element interactions. It's completely unnecessary.

The .shouldBeVisible() custom command isn't worth the trouble or abstraction when you can already use: .should('be.visible')"

Reply:
But still methods which check existence might be to abstract 🤔